spec/support/driver_registration.rb

```diff
@@ -1,19 +1,33 @@
 module DriverRegistration
-  def self.register_selenium_chrome
-    return if @selenium_chrome_registered
-    Capybara.register_driver :selenium_chrome do |app|
-      Capybara::Selenium::Driver.new(app, browser: :chrome)
-    end
-    Capybara::Screenshot.register_driver(:selenium_chrome) do |js_driver, path|
-      js_driver.browser.save_screenshot(path)
-    end
-    @selenium_chrome_registered = true
+  # def self.register_selenium_chrome
+  #   return if @selenium_chrome_registered
+  #   Capybara.register_driver :selenium_chrome do |app|
+  #     Capybara::Selenium::Driver.new(app, browser: :chrome)
+  #   end
+  #   Capybara::Screenshot.register_driver(:selenium_chrome) do |js_driver, path|
+  #     js_driver.browser.save_screenshot(path)
+  #   end
+  #   @selenium_chrome_registered = true
+  # end
+
+  def self.driver_caps(app)
```

Comment:
remove sauce

so sauce can be a small PR so people can reference it

nobody wants to reference a 6K line PR